linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_disk_full_action/ansible/shared.yml

```diff
@@ -8,7 +8,7 @@
 - name: Configure auditd Disk Full Action when Disk Space Is Full
   lineinfile:
     dest: /etc/audit/auditd.conf
-    line: "disk_full_action = {{ var_auditd_disk_full_action }}"
+    line: "disk_full_action = {{ var_auditd_disk_full_action.split('|')[0] }}"
```

Comment:
@jan-cerny Hi, the first value of `rhel8` selector is `syslog`, but the profile explicitly selected `halt`.
With this change, when remediating, the RHEL8 STIG profile will add `syslog` instead of `halt`. Is that an expected change?

CC: @Mab879 @ggbecker